### DISCUSSION TITLE
thought breaks in jats xml

### DISCUSSION THREAD

**Discussion by denismaier** [singlesourcepub/community#42] [General]
I need to typeset an article that uses asterisks to denote thought breaks. Any ideas how to tag that in JATS XML? Or where would be a good place to ask?

**Comment by gimsieke** [comment]
It’s best to ask on [JATS-List](https://www.mulberrytech.com/JATS/JATS-List/), the mailing list for all things JATS.

I don’t think there’s a special element available for such an [“asterism”](https://english.stackexchange.com/questions/40379/is-there-a-proper-name-for-the-3-asterisks-that-are-used-to-suggest-temporal-dis), so maybe use `p` with a specific `@content-type`, for example, `<p content-type="asterism">* * *</p>`.

**Reply by gimsieke** [reply]
Ah, it’s called [dinkus](https://en.wikipedia.org/wiki/Dinkus). An [asterism](https://en.wikipedia.org/wiki/Asterism_(typography)) looks like this: ⁂

**Comment by denismaier** [comment]
Thanks!
